.github/workflows/release.yaml

```
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: MIT
+
+name: release
+
+on:
+  push:
+    # run only against tags
+    tags:
+      - 'v*'
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+        with:
+          fetch-depth: 0
+
+      - run: git fetch --force --tags
+      - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
+        with:
+          go-version: '1.23.1'
```

Suggestion to change this to use `go.mod`'s version (I'm okay with doing this in a follow-up PR).

```suggestion
          go-version-file: go.mod
```

---

oh, didn't know that this is a thing, thank you